Makefile

```diff
@@ -274,15 +284,15 @@ install_cluster:
 _deploy_nodes:
 	discovery-infra/start_discovery.py -i $(ISO) -n $(NUM_MASTERS) -p $(STORAGE_POOL_PATH) -k '$(SSH_PUB_KEY)' -md $(MASTER_DISK) -wd $(WORKER_DISK) -mm $(MASTER_MEMORY) -wm $(WORKER_MEMORY) -nw $(NUM_WORKERS) -ps '$(PULL_SECRET)' -bd $(BASE_DOMAIN) -cN $(CLUSTER_NAME) -vN $(NETWORK_CIDR) -nM $(NETWORK_MTU) -iU $(REMOTE_SERVICE_URL) -id $(CLUSTER_ID) -mD $(BASE_DNS_DOMAINS) -ns $(NAMESPACE) -pX $(HTTP_PROXY_URL) -sX $(HTTPS_PROXY_URL) -nX $(NO_PROXY_VALUES) --service-name $(SERVICE_NAME) --vip-dhcp-allocation $(VIP_DHCP_ALLOCATION) --profile $(PROFILE) --ns-index $(NAMESPACE_INDEX) --deploy-target $(DEPLOY_TARGET) $(DAY1_PARAMS) $(OC_PARAMS) $(KEEP_ISO_FLAG) $(ADDITIONAL_PARAMS) $(DAY2_PARAMS) -ndw $(NUM_DAY2_WORKERS) --ipv4 $(IPv4) --ipv6 $(IPv6) --platform $(PLATFORM) --proxy $(PROXY) --iso-image-type $(ISO_IMAGE_TYPE)
 
-deploy_nodes_with_install:
+deploy_nodes_with_install: start_load_balancer
```

Comment:
we need load balancer only in ipv6 none platform, why to start it every run?

Reply:
We work with it also with IPv4

Reply:
Also, we need to run it outside skipper.  So this is the place since any podman we will run in skipper will be terminated when skipper terminates.  So this is the place to run it.  At this point, we don't know if we will need it.

Reply:
not sure i understand why we need it in regular flow? what it gives for us?

Reply:
so it is relevant only for None platform, lets spawn it there.

Reply:
if container is started as daemon and with nsenter? still it is killed when skipper exits?